src/server/identity/cmds/cmds.go

```diff
@@ -1,11 +1,18 @@
 package cmds
 
 import (
+	"bytes"
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
 	"os"
 
+	"github.com/gogo/protobuf/jsonpb"
+
+	"github.com/pachyderm/pachyderm/v2/src/server/identityutil"
```

Comment:
super nit, but I think your editor is reformatting the imports

Reply:
yeah that's a little weird, I can fix that though